src/Validation/ItValidation.php

```diff
@@ -88,15 +88,55 @@ public static function cf($check)
         return (chr($sum % 26 + ord('A')) == $check[15]);
     }
 
+    /**
+     * Checks Partita IVA, VAT number for Italy.
+     *
+     * @param string $check The value to check.
+     * @return bool Success.
+     */
+    public static function piva($check)
```

Comment:
Should this really use a localized method name?

Reply:
As long as it is not something very specific, maybe not?
Do most countries have such a number? Is the German Umsatzsteuer ID the same (I think so).

Reply:
Polish tax id number (NIP) is validated as a `personId` (which is wrong by the way, it's not a person ID, it's a number given to companies). Other numbers (pesel - actual person ID, regon) are localized.

Reply:
@inoas Yes, the german equivalent is the Umsatzsteuer-Identifikationsnummer. VAT IDs exist all around the world, hence I'd suggest to use a common english name like `vatId`.

Reply:
This should then also come with a test case, I think.